.github/dependabot.yaml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Please add the prefix `build` to all the `package-ecosystem` and include the `scope`.

```yml
commit-message:
  prefix: "build"
  include: "scope"
```

[Reference](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message)